aquasec-server/values.yaml

```diff
@@ -0,0 +1,132 @@
+# Specifies the secret data for imagePullSecrets needed to fetch the private docker images
+imageCredentials:
+  repositoryUriPrefix: "docker.io"
+  registry: "index.docker.io/v1/" #for docker.io #REQUIRED
+  username: #REQUIRED
+  password: #REQUIRED
+  email:
+
+rbac:
+  enabled: true
+  roleRef:
+
+db:
+  external:
+    enabled: false
+    name:
+    host:
+    port:
+    user:
+    password:
+    auditName:
+    auditHost:
+    auditPort:
+    auditUser:
+    auditPassword:
+  image:
+    repository: aquasec/database
+    tag: 3.2
+    pullPolicy: IfNotPresent
+  service:
+    type: ClusterIP
+  persistence:
+    enabled: true
+    storageClass:
+    size: 30Gi
+    accessMode: ReadWriteOnce
+  livenessProbe:
+    exec:
+      command:
+      - sh
+      - -c
+      - exec pg_isready --host $POD_IP
+    initialDelaySeconds: 60
+    timeoutSeconds: 5
+    failureThreshold: 6
+  readinessProbe:
+    exec:
+      command:
+      - sh
+      - -c
+      - exec pg_isready --host $POD_IP
+    initialDelaySeconds: 5
+    timeoutSeconds: 3
+    periodSeconds: 5
+  # TODO: Specify the minimal DB resource requirements
+  resources: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+gate:
+  image:
+    repository: aquasec/gateway
+    tag: 3.2
+    pullPolicy: IfNotPresent
+  service:
+    type: ClusterIP
+    externalPort: 3622
+  replicaCount: 1
+  # TODO: Fill-in the correct liveness and readiness probes
+  livenessProbe: {}
+  readinessProbe: {}
+  resources:
+    requests:
+      cpu: 0.1
+      memory: 0.2Gi
+    limits:
+      cpu: 1
+      memory: 2Gi
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+web:
+  image:
+    repository: aquasec/server
+    tag: 3.2
+    pullPolicy: IfNotPresent
+  service:
+    type: ClusterIP
+    externalPort: 8080
+  ingress:
+    enabled: false
+    annotations: {}
+    #  kubernetes.io/ingress.class: nginx
+    hosts: #REQUIRED
+    #- aquasec-test.example.com
+    tls: []
+    #  - secretName: aquasec-tls
+    #    hosts:
+    #      - aquasec.domain.com
+  replicaCount: 1
+  livenessProbe: {}
+  readinessProbe: {}
+  resources:
+    requests:
+      cpu: 0.1
+      memory: 0.2Gi
+    limits:
+      cpu: 1
+      memory: 2Gi
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+scanner:
+  enabled: false
+  image:
+    repository: aquasec/scanner-cli
+    tag: 3.2
+    pullPolicy: IfNotPresent
+  # TODO: Are there any defaults for the following values?
+  user:
+  password:
+  replicaCount: 1
+  # TODO: Fill-in the correct liveness and readiness probes and basic resource configuration
```

Comment:
@oranmoshai can you please provide the probes for scanner too

Reply:
Currently there is no probe for scanner we will add it to the roadmap